components/button/03-button--tertiary.hbs

```diff
@@ -0,0 +1 @@
+<button class="btn btn--tertiary">Print this page <i class="btn__icon btn__icon--print-link"></i></button>
```

Comment:
Agree with making the HTML semantic. `span` tags would be preferable for accessibility purposes.

I’ve decided not to use the pseudo elements with a modifier on the `.btn` block to draw the icons—it wouldn’t be possible to use both a modifier for a button icon and `.button--loader` background on a single element at the same time. The icon implementation already uses before/after and would be better to give it a separate element for its single responsibility.

Might be easier to think a button has an icon rather than a button is an icon.
Will go ahead and make the accessibility related changes.